prov/efa/src/efa_base_ep.h

```diff
@@ -56,6 +56,8 @@ struct efa_base_ep {
 	struct ibv_recv_wr recv_more_wr_head;
 	struct ibv_recv_wr *recv_more_wr_tail;
 	struct efa_recv_wr *efa_recv_wr_vec;
+	struct efa_recv_wr *user_recv_wr_vec;
+	size_t recv_wr_index;
```

Comment:
nit: `recv_wr_index` -> `user_recv_wr_index` and move these two fields after L63

Reply:
hmmm you will have `recv_wr_index` for efa-raw as well. So nvm, `user_recv_wr_vec` can be moved after L63.